packages/terra-table/src/Table.jsx

```diff
@@ -101,6 +101,11 @@ const propTypes = {
    * rendered to allow for row selection to occur.
    */
   hasSelectableRows: PropTypes.bool,
+
+  /**
+   * Boolean indicating whether or not the table should have zebra striping for rows.
```

Comment:
```suggestion
   * Boolean specifying whether or not the table should have zebra striping for rows.
```
I know we use "indicating" in some places, but "specifying" or "defining" is more accurate.